src/com/cft/contactmerge/io/SplitStringUtilities.java

```diff
@@ -10,23 +10,18 @@
 import java.util.*;
 
 public class SplitStringUtilities {
-    public static List<String> splitTsvString(String input)
-    {
+    public static List<String> splitTsvString(String input) {
         List<String> x = new ArrayList<String>();
 
-
-
         String testString = input;
 
-
         String _string = new String("");
         for(int i = 0;i< testString.length();i++){
             if(input.charAt(i)=='\t'){
                 x.add(_string);
                 _string = "";
             }
-            else
-            {
+            else {
                 _string= _string+input.charAt(i);
             }
         }
```

Comment:
I think a cleaner solution instead of writing a for loop for this would be `List<String> x = input.split("\t")`